doc/requirements.txt

```diff
@@ -3,9 +3,13 @@ appdirs
 autograd
 numpy
 pygments-github-lexers
+git+git://github.com/XanaduAI/pennylane-sf.git#egg=PennyLane_SF
+pennylane-forest
+pyquil
 scipy
 sphinxcontrib-bibtex
 sphinx_gallery
+strawberryfields
```

Comment:
actually, how come these are now needed? Is it because the scripts are auto-run?

Reply:
Yep